src/plots/geo/layout_attributes.js

```diff
@@ -174,9 +174,31 @@ var attrs = module.exports = overrideAll({
             min: 0,
             dflt: 1,
             description: [
-                'Zooms in or out on the map view.',
-                'A scale of *1* corresponds to the largest zoom level',
-                'that fits the map\'s lon and lat ranges. '
+                'Initial zoom level of the map view.',
+                'A scale of *1* (100%) corresponds to the base zoom level',
+                'that fits the map\'s lon and lat ranges.'
+            ].join(' ')
+        },
+        minScale: {
```

Comment:
Thanks for the PR.
Would you please use lowerscale for the name of attributes i.e. `minsale` and `maxscale`?

Reply:
fixed with c9f02d84746ae85d9f45b6fb7bd70996af03af55